onnxruntime/contrib_ops/cuda/bert/attention.cc

```diff
@@ -89,13 +89,14 @@ Status Attention<T>::ComputeInternal(OpKernelContext* context) const {
   // Check whether we can use fused kernel
   int sm = device_prop.major * 10 + device_prop.minor;
   bool is_mask_1d_seq_len = parameters.mask_type == AttentionMaskType::MASK_1D_KEY_SEQ_LEN;
+  bool is_mask_2d_key_padding = parameters.mask_type == AttentionMaskType::MASK_2D_KEY_PADDING;
 
   if (is_unidirectional_) {  // GPT
     // Fused kernels requires left side padding (The mask shall be sequence lengths or no mask)
```

Comment:
Update comment: Fused kernels requires one-side padding: mask can be non (no padding), 1D sequence lengths (right side padding) or 2d mask (either left or right-side padding).

Reply:
done